src/App.js

```diff
@@ -1,10 +1,27 @@
-import logo from './logo.svg';
+import { Routes, Route } from "react-router-dom";
 import './App.css';
 
+import {Home} from "./pages/home/home.jsx";
+import {Cart} from "./pages/cart/cart.jsx";
+import {WishList} from "./pages/wish-list/wish-list.jsx";
+import {ProductListing} from "./pages/product-listing/product-listing.jsx";
+import {Nav} from "./components/nav.jsx";
+import {Login} from "./pages/auth/login.jsx";
+import {SignUp} from "./pages/auth/sign-up.jsx"
+
```

Comment:
You can use an ```index.js``` file and add all your exports here and you can simply import all your component from a single file.